### PR TITLE
fix viewsource for plist file format

### PR DESCRIPTION
**Pull request by magaofei**
Plist file will be garbled if it is read directly

### CODE REVIEW

**Comment by ajinabraham** [comment]
@magaofei what happens if the plist file is not a binary plist?

File: StaticAnalyzer/views/ios/view_source.py [hunk 1]
```diff
@@ -34,7 +35,9 @@ def set_ext_api(file_path):
     """
     ext = file_path.split('.')[-1]
     if ext == "plist":
-        return "xml"
+        return "plist"
```
Comment (ajinabraham) [inline StaticAnalyzer/views/ios/view_source.py]: :TODO : check if `xml` has to do something with Syntax highlighting.

File: StaticAnalyzer/views/ios/view_source.py [hunk 2]
```diff
@@ -84,6 +87,9 @@ def run(request, api=False):
             file_format = 'xml'
             with io.open(sfile, mode='r', encoding="utf8", errors="ignore") as flip:
                 dat = flip.read()
+        elif typ == 'plist':
+            file_format = 'plist'
+            dat = biplist.readPlist(sfile)
```
Comment (ajinabraham) [inline StaticAnalyzer/views/ios/view_source.py]: What happens if it reads a non binary plist. Like from an iOS Source Zip

**Comment by magaofei** [comment]
Just like Android's Manifest will be specially processed, iOS plist files are also needed, and can not be read directly like xml files. In my tests, some files can be read directly, but more files will be garbled.

**Pull request merged**